Composer/packages/extensions/adaptive-flow/src/adaptive-flow-editor/hooks/useEditorEventApi.ts

```diff
@@ -80,6 +81,47 @@ export const useEditorEventApi = (
           announce(ScreenReaderMessage.ActionFocused);
         };
         break;
+      case NodeEventTypes.CtrlClick:
+        handler = (e: { id: string; tab?: string }) => {
+          if (!focusedId && !selectedIds.length) {
+            return handleEditorEvent(NodeEventTypes.Focus, e);
+          }
+
+          // Toggle the selection state of clicked id
+          const alreadySelected = selectedIds.some((x) => x === e.id);
+          if (alreadySelected) {
+            const shrinkedSelection = selectedIds.filter((x) => x !== e.id);
+            setSelectedIds(shrinkedSelection);
+            if (focusedId === e.id) {
+              onFocusSteps([shrinkedSelection[0] || '']);
+            }
+            announce(ScreenReaderMessage.ActionUnfocused);
```

Comment:
Is there a way to get the title of the action into these messages? I think it would make for better feedback.

Reply:
This module is blind to the action title, it only interacts with action's id.
If this is an important requirement, we can track it in a separated issue aims at unifing all existing operations' messages.

Reply:
I won't block this particular PR on this issue, but I ran into this a lot fixing a11y bugs - there's no good way to get the title of an action, and it looks like the same problem is happening here. I think we need to do what you said, just to have some internal way that those of us working on the editor can reliably get an action's title for use in announcements and things.

Reply:
@beyackle Yeah, I do remember you wrote some code to query DOM properties.
The action title could be a static map in which each $kind has a constant title string. It would be a good time to do that during the multi locale work. We create multiple title sets for different locales, then announcements could carry title info. What's your opinion?